doc/plotting/index.rst

```diff
@@ -0,0 +1,20 @@
+.. include:: ../tune_toc.rst
+
+
+.. _plotting:
+
+=========================
+Visualization
+=========================
+
+In this section, we detail the general tools to visualize
+neuroimaging volumes with nilearn.
+
+|
+
+.. include:: ../includes/big_toc_css.rst
+
+
+.. toctree::
+
+   plotting.rst
```

Comment:
To make sure myself, this means that should move content from plotting.rst here and remove both name and file plotting.rst and tune_toc.rst ?